cypress/integration/common/search.js

```diff
@@ -63,3 +63,11 @@ Then('the page contains meta tags with the following names', dataTable => {
     cy.get(locator).should('have.attr', 'content').and('be.eq', content);
   }
 });
+
+Then('the user is redirected to {string}', (redirectUrl) => {
+  cy.location('pathname').should('include', redirectUrl);
+});
+
+Then('the search bar on the page does not maintain the user’s term', () => {
+  cy.get('#keywords').should('have.attr','placeholder').and('be.eq','Enter keywords or phrases');
```

Comment:
The placeholder will always be equal to 'Enter keywords or phrases', whether it is displayed or not. You should check the attribute 'value' instead and assert that its empty

Reply:
This makes sense - though there are many other places that we have that placeholder text test. Maybe we should have a ticket to clean those up in the future? But this particular instance has been updated in the recently-pushed commit!

Reply:
Those instances with `placeholder` text would only be when we are actually checking for `placeholder` values. For this test case scenario we are checking the `input` value which is different.